.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: ROS 2 CI
+on:
```

Comment:
Would you like scheduled runs too? Like say nightly runs

Reply:
I think this is a good idea, but for the first PR I will keep it simple and I will iterate in the CI jobs in a second one.